airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml

```diff
@@ -44,6 +44,9 @@ data:
               - "ads_insights_region"
               - "ads_insights_dma"
               - "ads_insights_action_product_id"
+      3.0.0:
```

Comment:
What makes it breaking if we have the config migration? Does that mean that the user will have to opt-in on that change?

Reply:
no, I was just guessing this because the PR cannot be directly reverted. I can do only a 2.2.0 version increment. WDYT?

Reply:
I think 2.2.0 makes sense, yes! We should still document that the config format changes but there is a migration for backward compatibility but none for forward compatiblity

Reply:
great. Thanks Max!